EIPS/eip-4750.md

```diff
@@ -2,7 +2,7 @@
 eip: 4750
 title: EOF - Functions
 description: Individual sections for functions with `CALLF` and `RETF` instructions
-author: Andrei Maiboroda (@gumb0), Alex Beregszaszi (@axic), Paweł Bylica (@chfast)
+author: Andrei Maiboroda (@gumb0), Alex Beregszaszi (@axic), Paweł Bylica (@chfast), Greg Colvin, Brooklyn Zelenka (@expede)
```

Comment:
CC'ing existing authors: @gumb0 @axic @chfast
CC'ing potential new authors: @gcolvin @expede
Can you all confirm you are okay with this change?

Reply:
Personally, I'm generally fine with one existing author approving (don't need all of them).  I tend to agree that the authors being added should all agree.

Reply:
> one existing author approving
I would prefer half, rounded up.

Reply:
I would prefer these changes to be a separate EIP, requiring this one.

Reply:
I put this up here so that we could have a precise proposal to discuss.  I don't expect it to be merged in its present form.